wurst/systems/commands/Skins.wurst

```diff
@@ -86,8 +85,14 @@ function onEnterHandler(unit target)
     // Register the command used to switch to the custom skin.
     registerCommand(target.getOwner(), skin.featureName) (owner, args) ->
         // Transform the unit to impart the custom skin.
-        // TODO: Generate skins to match trolls and uncomment this.
-        // transformUnit(target, skin.newUnitTypeId)
+        BlzSetUnitSkin(target, skin.newUnitTypeId)
+
+        // For some reason the hero glow disappear on skin switch
+        // but the unit still has the ability, resetting it to "re-enable" the glow
+        // Master Healer skin already has hero glow
+        if skin.newUnitTypeId != UNIT_MASTER_HEALER_1
```

Comment:
Can you check for whether the unit has the ability directly, e.g. `if target.hasAbility(ABILITY_HERO_GLOW)`? Checking type ID directly is fragile.